### PR TITLE
Add turborepo to monorepo.

### PR DESCRIPTION
**Pull request by leerob**
```bash
▲  documenso/ turbo run build            
• Packages in scope: @documenso/features, @documenso/lib, @documenso/pdf, @documenso/prisma, @documenso/signing, @documenso/ui, @documenso/web
• Running build in 7 packages

...
...
...

 Tasks:    1 successful, 1 total
Cached:    1 cached, 1 total
  Time:    156ms >>> FULL TURBO
```

### CODE REVIEW

**Comment by adithyaakrishna** [comment]
@leerob This is super amazing! I absolutely love how the whole dev community is coming together to improve and add to this product 🚀

File: turbo.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": [".env"],
```
Comment (leerob) [inline turbo.json:3]: We probably want to instead explicitly list all of the env vars here 👍

**Review by PeerRich — DISMISSED** [review]
hell yea-- turbofied

**Review by dephraiim — APPROVED** [review]

File: turbo.json (hunk 1 above)
Comment (leerob) [reply turbo.json]: Pushed an update for this: https://turbo.build/repo/docs/reference/configuration#globalenv

**Comment by Mythie** [comment]
> @leerob This is super amazing! I absolutely love how the whole dev community is coming together to improve and add to this product 🚀

I agree, it's awesome to so many people dropping by our repo 😄

**Review by Mythie — APPROVED** [review]

**Review by nikospapcom — APPROVED** [review]

**Comment by Mythie** [comment]
Thanks for the fantastic first contribution Lee!

just-wanted-to-say-thanks

**Pull request merged**